apps/emqx_bridge_gcp_pubsub/src/emqx_bridge_gcp_pubsub_impl_consumer.erl

```diff
@@ -426,3 +437,18 @@ log_when_error(Fun, Log) ->
 
 forget_interval(infinity) -> ?DEFAULT_FORGET_INTERVAL;
 forget_interval(Timeout) -> 2 * Timeout.
+
+-spec get_transport() -> {tls | tcp, string()}.
+get_transport() ->
+    %% emulating the emulator behavior
+    %% https://cloud.google.com/pubsub/docs/emulator
+    case os:getenv("PUBSUB_EMULATOR_HOST") of
+        false ->
+            {tls, "pubsub.googleapis.com:443"};
+        HostPort0 ->
+            %% The emulator is plain HTTP...
+            Transport0 = persistent_term:get(
```

Comment:
Maybe use `emqx_bridge_gcp_pubsub_client`'s function to avoid fetching private trerm?

Reply:
Actually, this is only used in tests, and the thing to be fetched depends on the caller (if it's pubsub or bigquery). :thinking: